src/components/Class/index.js

```diff
@@ -62,13 +63,27 @@ function Class() {
     event && event.preventDefault();
     const formData = new FormData(event.target);
     const formFields = {};
+
+    const obj = {
```

Comment:
The better name could be. `languageMap`? what do you think? And did you try to create a new class after making this change?

Reply:
The variable name has been changed. Yes, I checked it's working fine